suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic } from '../../utils/ceramicHelpers.js'
+import { createDid, splitDidsToList } from '../../utils/didHelper.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import {Model} from '@ceramicnetwork/stream-model'
+import {ModelInstanceDocument} from '@ceramicnetwork/stream-model-instance'
+import { newModel, basicModelDocumentContent } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+// TODO : Update to the modelId created manually on all envs, if id cannot be the same then create an array for different envs  
+let modelId = "kjzl6hvfrbw6c59m8k5l50gqmsduszfazp25wakaep3nsiyieff7dbdvpgb0ypf"
+let seed = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS)
+const magicNumber = 5000;
+describe('Model Integration Test', () => {
+  let ceramicNode1: CeramicClient
+  let ceramicNode2: CeramicClient
+  let streamId = StreamID.fromString(modelId);
+  beforeAll(async () => {
+    let did = await createDid(splitDidsToList(seed)[1]);
+    ceramicNode1 = await newCeramic(ComposeDbUrls[0], did)
+    ceramicNode2 = await newCeramic(ComposeDbUrls[1], did)
+    const indexedModels = await ceramicNode1.admin.getIndexedModels();
+    if(!indexedModels.includes(streamId)){
+        let model = await Model.create(ceramicNode1, newModel);
+        streamId = model.id;
+        console.log("Model not found on node 1 creating: ", model)
+        TestUtils.waitForConditionOrTimeout(async () =>
+            ceramicNode1
+                    .loadStream(model.id)
+                    .then((_) => true)
+                    .catch((_) => false)
+        )
+        await ceramicNode1.admin.startIndexingModels([streamId]);
```

Comment:
Why is the order of operations different between node1 and node2.

On node1 we wait for the load stream to finish and then start indexing the model. On node2 we start indexing the model and then wait for the load stream to finish.

Can they be the same?